apps/platform-app/prisma/migrations/20241105184615_add_approval_modal_shown/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "Profile" ADD COLUMN     "wasApprovalModalShown" BOOLEAN NOT NULL DEFAULT false;
```

Comment:
**suggestion**: Consider adding a comment explaining the purpose of the `wasApprovalModalShown` column for future reference.